metro.config.js

```diff
@@ -1,9 +1,9 @@
-const { getDefaultConfig } = require("metro-config");
+const { getDefaultConfig } = require("expo/metro-config");
 
 module.exports = (async () => {
   const {
     resolver: { sourceExts, assetExts },
-  } = await getDefaultConfig();
+  } = await getDefaultConfig(__dirname);
```

Comment:
why do we need this now?

Reply:
https://docs.expo.dev/guides/customizing-metro/ this is how they do it in their example leh, when i try without 2 sprints ago it doesnt work